app/controllers/collections_controller.rb

```diff
@@ -5,7 +5,7 @@
 class CollectionsController < ObjectsController
   before_action :authenticate_user!
   before_action :ensure_sdr_updatable
-  verify_authorized
+  verify_authorized except: [:show]
```

Comment:
Why are we not authorizing the show method?

Reply:
This matches what we have in the WorksController... Without this I'm getting an unauthorized error.

Would our preference be to add a policy? Though I'm not seeing one for works either.

Reply:
Yes, we should have policies.  We do not want these show pages to be accessible to the general population.